src/Filter/FilterFunction.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Filter;
+
+use Assert\Assertion;
+use Lmc\ApiFilter\Entity\Value;
+
+class FilterFunction extends AbstractFilter
+{
+    public const TITLE = 'fun';
+
+    public function __construct(string $column, Value $value, ?string $title = null)
```

Comment:
Why not `string $title = self::TITLE`?

Reply:
because it is much easier to send `null` and use `defaultValue` - otherwise parser (and factory) would need to know, what is a default value and send it instead of `null`